packages/graphql/src/makeGraphQLServer.ts

```diff
@@ -109,6 +112,16 @@ export const graphQLHTTP = graphqlHTTP((req, res, params) => {
     schema: graphqlSchema,
     graphiql: SHOW_GRAPHIQL,
     context: ctx,
+    customExecuteFn: (args) => {
```

Comment:
is it worth not adding this if `!debugOperation.enabled`? it would save the creation of 2 Date objects per graphql opt for 99.9% of users not running with `cypress-verbose` enabled

Reply:
We could, thought it's also one of those things where saving two date objects might not be worth the complexity of having conditional codepaths relative to the performance compared to almost anything we do or could optimize in the system elsewhere. Micro-optimizations are a slippery slope.

Also was thinking that in the future we might do telemetry/performance metrics in places like this in all cases rather than conditionally.

I'd be fine either way.